Sources/Resources/Base.lproj/LaunchScreen.storyboard

```diff
@@ -16,18 +16,17 @@
                         <rect key="frame" x="0.0" y="0.0" width="393" height="852"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
-                            <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="logo" translatesAutoresizingMaskIntoConstraints="NO" id="T97-nu-3o4">
-                                <rect key="frame" x="143" y="374" width="108" height="105"/>
+                            <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="Tudor crown with GOV.UK" translatesAutoresizingMaskIntoConstraints="NO" id="T97-nu-3o4" userLabel="Tudor crown with GOV.UK">
+                                <rect key="frame" x="143" y="368.66666666666669" width="107" height="115"/>
```

Comment:
Figma design states: width: 107, height: 114, x: 134, y: 363. However to centre the image it need to be moved slightly